hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/actionhistory/ActionHistoryLayout.java

```diff
@@ -58,17 +61,31 @@ public ActionHistoryLayout(final VaadinMessageSource i18n, final DeploymentManag
         this.deploymentManagement = deploymentManagement;
         this.notification = notification;
         this.managementUIState = managementUIState;
+        actionHistoryCaption = getActionHistoryCaption();
         init();
     }
 
+    private final String getActionHistoryCaption() {
+        return getActionHistoryCaption(null);
+    }
+
+    private String getActionHistoryCaption(final String targetName) {
+        String caption = getI18n().getMessage("caption.action.history");
```

Comment:
"for" should not be hardcoded... I'd recomment to define two messages with two separate keys...